build-caching/src/main/java/com/vertispan/j2cl/build/DefaultDiskCache.java

```diff
@@ -72,4 +52,9 @@ protected Path logFile(Path taskDir) {
     protected Path outputDir(Path taskDir) {
         return taskDir.resolve("results");
     }
+
+    @Override
+    protected Path cacheSummary(Path taskDir) {
+        return taskDir.resolve("cacheSummary.json");
```

Comment:
Follow-up task for a reader for these files, and for more metadata about the build (like build times) to be written in the same format?

Reply:
Filed as #225.